# LaneWaypoints.msg
float32[] left_points
float32[] right_points
